src/app/routes/api/v3/admin/forms/admin-forms.settings.routes.ts

```diff
@@ -6,10 +6,10 @@ import { SettingsUpdateDto } from '../../../../../../types/api'
 import { withUserAuthentication } from '../../../../../modules/auth/auth.middlewares'
 import { handleUpdateSettings } from '../../../../../modules/form/admin-form/admin-form.controller'
 
-export const AdminFormsRouter = Router()
+export const AdminFormsSettingsRouter = Router()
 
 // All routes in this handler should be protected by authentication.
-AdminFormsRouter.use(withUserAuthentication)
+AdminFormsSettingsRouter.use(withUserAuthentication)
```

Comment:
that's the danger with splitting up the routes like this during refactors haha.

i suggest putting the `withUserAuthentication` middleware into the index router